docs/reference/objects/package.md

```diff
@@ -95,7 +95,6 @@ The state of the package. Can be one of:
 - draft
 - selling
 - off_sale
-
```

Comment:
Removed line is causing formatting issues

![CleanShot 2023-08-08 at 16 51 59](https://github.com/easolhq/easolhq.github.io/assets/48216064/2a685e27-f658-40ad-988d-9469141c9fcb)